src/client/sandbox/native-methods.js

```diff
@@ -193,9 +193,13 @@ class NativeMethods {
 
         if (win.MessageEvent) {
```

Comment:
Which of supported browsers did not have `MessageEvent`. Write comment about it.